tensorflow_addons/text/parse_time_op_test.py

```diff
@@ -0,0 +1,60 @@
+# Copyright 2019 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+"""Parse time op tests."""
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import tensorflow as tf
+
+from tensorflow_addons import text
+from tensorflow_addons.utils import test_utils
+
+
+@test_utils.run_all_in_graph_and_eager_modes
+class ParseTimeTest(tf.test.TestCase):
+    def test_parse_time(self):
+        time_format = "%Y-%m-%dT%H:%M:%E*S%Ez"
+        items = [
+            ("2019-05-17T23:56:09.05Z", time_format, "NANOSECOND",
+             1558137369050000000),
+            ("2019-05-17T23:56:09.05Z", time_format, "MICROSECOND",
+             1558137369050000),
+            ("2019-05-17T23:56:09.05Z", time_format, "MILLISECOND",
+             1558137369050),
+            ("2019-05-17T23:56:09.05Z", time_format, "SECOND", 1558137369),
+            ([
+                "2019-05-17T23:56:09.05Z", "2019-05-20T11:22:33.44Z",
+                "2019-05-30T22:33:44.55Z"
+            ], time_format, "MILLISECOND",
+             [1558137369050, 1558351353440, 1559255624550]),
+        ]
+        for e in items:
+            result = text.parse_time(
+                time_string=e[0], time_format=e[1], output_unit=e[2])
+            expected = e[3]
+
+            if not tf.executing_eagerly():
+                result = result.eval()
+
+            if isinstance(expected, list):
```

Comment:
Can we use assertAllEqual for both cases here?

Reply:
Thanks, assertAllEqual works!